File: 2980_CheckBitwiseORTrailingZeroes/solution.cs
public class Solution {
    public bool HasTrailingZeros(int[] nums) {
        int ORbin = 0;
        for(int i = 0; i < nums.Length; i++){
            if(nums[i] % 2 == 0){
                ORbin++;
            }
            if(ORbin == 2){
                return true;
            }
        }
        return false;
    }
}

File: 2980_CheckBitwiseORTrailingZeroes/solution.py
class Solution(object):
    def hasTrailingZeros(self, nums):
        ORbin = 0
        for i in range(len(nums)):
            if(nums[i] % 2 == 0):
                ORbin += 1
            if(ORbin == 2):
                return True
        return False
        """
        :type nums: List[int]
        :rtype: bool
        """
        
